MMM-WeatherChart.js

```diff
@@ -42,6 +42,10 @@ Module.register("MMM-WeatherChart", {
         showSnow: false,
         showZeroSnow: true,
         color: 'rgba(255, 255, 255, 1)',
+        colorMin: 'yellow',
```

Comment:
The default value for the new color options should be 'rgba(255, 255, 255, 1)' for compatibility.

Reply:
Changed the default and added the colors as example in the documentation.